server/ingester/flow_log/config/config.go

```diff
@@ -37,7 +37,7 @@ const (
 )
 
 var DefaultOtlpExportDatas = []string{"cbpf-net-span", "ebpf-sys-span"}
-var DefaultOtlpExportDataTypes = []string{"service_info", "tracing_info", "network_layer", "flow_info", "transport_layer", "application_layer", "metrics"}
+var DefaultOtlpExportDataTypes = []string{"service_info", "tracing_info", "network_layer", "flow_info", "transport_layer", "application_layer", "metrics", "client_custom_tag", "server_custom_tag"}
```

Comment:
whether **client/server custom tag** can be merged into one config.

Reply:
> whether **client/server custom tag** can be merged into one config ?

The benefit of the merger is that users are not required to understand some non-critical concepts.